vm/actor/src/builtin/cron/mod.rs

```diff
@@ -74,12 +74,12 @@ impl Actor {
 
         let st: State = rt.state()?;
         for entry in st.entries {
-            rt.send(
+            let _v = rt.send(
                 &entry.receiver,
                 entry.method_num,
                 &Serialized::default(),
                 &TokenAmount::from(0u8),
-            )?;
+            );
```

Comment:
revert back to what it was, this is ignoring that error

Reply:
Yeah I was confused by this too. In the spec-actors repo they ignore any error or return values. I wasn't sure about why they did it but I just mimicked what they had. Are they doing something wrong?

Reply:
ohh, you're right. Interesting why that error is ignored, but I guess it makes sense. Possibly add a comment here to indicate that this is intentional?

Reply:
I would do `let _ =` instead of `let _v =` but yeah, so weird they dont use the error...